### ISSUE TITLE
Feature: Tetarea Auto resize

### ISSUE THREAD

**Issue by kristof12345** [SamProf/MatBlazor#571]
Hi!

It'd be great to have an option to enable auto resize for Textarea. If you add a line, the height automatically adjusts. (In this mode the scrollbars should be also hidden.)

**Comment by enkodellc** [comment]
Have you seen this done elsewhere?  Maybe submit a PR for it .  "It'd be great" to have more contributors :)

**Comment by kristof12345** [comment]
Yeah, the React version does this. https://material-ui.com/components/text-fields/ (See the "multiline" part.)

Through js it can be done with jquery: https://css-tricks.com/textarea-tricks/
But I don't know if it should be used with Blazor.
